demos/can-stache-bindings/event-args.html

```diff
@@ -1,6 +1,6 @@
 <script type='text/stache' can-autorender id='demo-html'>
-{{#each(items)}}
-	<li on:click='items.splice(scope.index,1)'>{{name}}</li>
+{{#each(items, item=value num=index)}}
+	<li on:click='../items.splice(num,1)'>{{item.name}}</li>
```

Comment:
I don’t know if it’s a bug or not, but `scope.index` can’t be used because it’ll be the last value after going through the entire list.

Reply:
You mean it's always set to `items.length - 1`?

Reply:
If so, that's a bug.